docs/source/testing/android-studio-plugin.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: Android Studio / IntelliJ plugin
+---
+
+To help your day-to-day development with Apollo Kotlin, a plugin for Android Studio and IntelliJ is available.
+
+## Installation
+
+The plugin is available on the JetBrains plugin repository. To install it from the IDE, go to
+<kbd>Settings</kbd> | <kbd>Plugins</kbd> | <kbd>Marketplace</kbd> | search for "Apollo GraphQL" | <kbd>Install</kbd>
+
+The plugin is supported on:
+
+- IntelliJ 2022.2 and above
+- Android Studio 2022.2 (Flamingo) and above
+
+## Features
+
+### Automatic code generation
+
+The plugin automatically generates models whenever GraphQL files change.
+This means you don't need to manually run `./gradlew generateApolloSources` after editing the files.
+
+Note: this works by running Gradle in continuous mode, and can be disabled in the plugin settings.
+
+### Integration with the GraphQL IntelliJ Plugin
+
+The plugin depends on and integrates with the [GraphQL IntelliJ Plugin](https://plugins.jetbrains.com/plugin/8097-js-graphql).
+The file structure of the Apollo project is automatically contributed, so there is no need to create a `graphql.config.yml` / `.graphqlconfig` file.
+
+### Navigation
+
+#### Kotlin → GraphQL
+
+When editing Kotlin code, <img src="images/gutter-icon.png" alt="Gutter icon" width="16" valign="top"/> icons will show next to references to generated operation and fragment classes.
```

Comment:
Inline images do not work :(. I'm no css expert but maybe a `display: inline-block;` would work here?

![Screenshot 2023-07-25 at 13 49 30](https://github.com/apollographql/apollo-kotlin/assets/3974977/b24074a5-750d-4b49-9137-48e519c0ec26)